src/main/kotlin/no/nav/familie/ef/mottak/service/TaskService.kt

```diff
@@ -24,5 +25,15 @@ class TaskService(private val taskRepository: TaskRepository,
         soknadRepository.save(it.copy(taskOpprettet = true))
     }
 
+    @Transactional
+    fun sendSøknadMottattTilDittNavTask(it: Soknad) {
+        val properties =
+                Properties().apply { this["søkersFødselsnummer"] = it.fnr }.apply { this["dokumenttype"] = it.dokumenttype }
+        taskRepository.save(Task.nyTask(SEND_SØKNAD_MOTTATT_TIL_DITT_NAV,
+                                        it.id,
+                                        properties))
+        soknadRepository.save(it.copy(taskOpprettet = true))
```

Comment:
Søknaden blir lagret 2 ganger? Her og i opprettPdfTaskForSoknad

Reply:
ups! Bra sett. Endrer it -> søknad.

Reply:
Kunne opprettet send til ditt nav i samme metode? Hva tror du om det?

Reply:
(evt. onComplete i pdf?)

Reply:
Eller att taskService har en metode som er opprett task, og att den oppretter de 2 ulike taskene
Nå er det litt skjult att en av de oppdaterer soknaden. Muligens det burde være en test som verifiserer

Alternativt att man har en test som verifiserer att soknad er lagret en gang
`verify (exactly = 1) { soknadRepository.save(any()) }`

Reply:
Flyttet dittNav task til samme metode som opprett pdfTask. Nytt navn: startTaskProsessering

Dekket av test her: 
no.nav.familie.ef.mottak.service.TaskServiceTest#opprettTask oppretter en task for søknad og setter taskOpprettet på søknaden til true
-> assertThat(soknadSlot.captured.taskOpprettet).isTrue